content/millennium/r4/base/individuals/patient.md

```diff
@@ -80,6 +81,9 @@ Notes:
   * May be provided twice using the prefixes `le` and `ge` to indicate a date range
   * May be provided once using one of the following prefixes to imply a date range: `ge`, `le`, `gt`, `lt`, `eq`
   * Must not be provided with a time component
+* The `_revinclude` parameter may be provided once with the value `Provenance:target`. Example: `_revinclude=Provenance:target`
+* The `_revinclude` parameter may be provided with the `_id` parameter. Example: `_id=629928&_revinclude=Provenance:target`
```

Comment:
This information convey like _revinclude can be provided with only _id.
Need to change this